mapswipe_workers/mapswipe_workers/firebase_to_postgres/delete_project.py

```diff
@@ -0,0 +1,34 @@
+"""
+Delete projects.
+"""
+
+from mapswipe_workers import auth
+from mapswipe_workers.definitions import logger
+
+
+def delete_project(project_ids: list) -> None:
+    """
+    Deletes projects.
+
+    Deletes project, groups, tasks and results from Firebase and Postgres
+    """
+    for project_id in project_ids:
+        logger.info(
+            f"Delete project, groups, tasks and results of project: {project_id}"
+        )
+
+        fb_db = auth.firebaseDB()
+        fb_db.reference(f"v2/results/{project_id}").delete()
+        fb_db.reference(f"v2/tasks/{project_id}").delete()
```

Comment:
similar to the approach in archive projects we might need to think about splitting this up into chunks to avoid write size limit errors

Reply:
Does this limit also apply to the delete function? I will try to find this out.

Reply:
The limit also apllies to the delete functiion. I could test that. A soloution using chunks of data is now implemented (see also #310).